examples/network-show/id-entity.out.static

```diff
@@ -0,0 +1,14 @@
+=== ENTITY ===
+Entity Address: oasis1qzp84num6xgspdst65yv7yqegln6ndcxmuuq8s9w
+Entity ID:      xQN6ffLSdc51EfEQ2BzltK1iWYAw6Y1CkBAbFzlhhEQ=
+Stake:          11601040.379679788 ROSE
+Commission:     20.0%
+
+=== NODES ===
+Node ID: Kb6opWKGbJHL0LK2Lto+m5ROIAXLhIr1lxQz0/kAOUM=
+  Node Roles: validator
+  Software Version: 24.1
+  Node Status:
```

Comment:
Align these values (e.g. "validator" and "24.1" should start in the same position).